kasm-single-zone/values.yaml

```diff
@@ -29,7 +29,12 @@ global:
 
   ## @param global.hostname Global hostname to use for TLS Certificates and ingress URL
   ##
-  hostname: ""
+  hostname: "kasm.uds.dev"
```

Comment:
was this accidental?

Reply:
100%. Thanks for the catch

Reply:
Removed the `nginx` callout as well. Ingress class will be added if you supply as value other than the empty string, or skipped if nothing is added.